README.md

```diff
@@ -304,7 +304,7 @@ $ret.items.client_id | get-unique
 $ret.items.client_id | Get-GRRComputerNameFromClientId -Credential $cred | get-unique
 
 # Get unique file paths from a file finder hunt
-($ret.items.payload.stat_entry.aff4path).substring(31) | sort -u
+($ret.items.payload.stat_entry.pathspec.path).substring(31) | sort -u
```

Comment:
substring(31) should not be needed anymore because we filtered out the aff4 prefix using the substring function.

```suggestion
$ret.items.payload.stat_entry.pathspec.path | sort -u
```